proxy/http2/Http2Stream.cc

```diff
@@ -163,6 +164,8 @@ Http2Stream::send_request(Http2ConnectionState &cstate)
       block = request_buffer.get_current_block();
     }
     done = _req_header.print(block->start(), block->write_avail(), &bufindex, &tmp);
+    TraceIn(h2_trace, cstate.ua_session->get_netvc()->get_remote_addr(), cstate.ua_session->get_netvc()->get_remote_port(),
```

Comment:
yes it prints HTTP/1.1 format. Current tracing does not handle binary data very well. I think we can print the H2 format